src/app/register/components/form-personal-additional-emails/form-personal-additional-emails.component.html

```diff
@@ -0,0 +1,101 @@
+<!-- ADDITIONAL EMAILS -->
+
+<ng-container [formGroup]="additionalEmails">
+  <div
+    class="input-container"
+    *ngFor="
+      let additionalEmailControl of additionalEmails.controls | keyvalue;
+      index as i
+    "
+  >
+    <mat-form-field appearance="outline">
+      <mat-label>
+        <ng-container i18n="@@register.additionalEmails"
+          >Additional email</ng-container
+        >
+        {{
+          additionalEmailControl.key !== '0'
+            ? parseInt(additionalEmailControl.key)
+            : ''
+        }}
+        <ng-container i18n="@@register.optional">(Optional)</ng-container>
+      </mat-label>
+      <input
+        formControlName="{{ additionalEmailControl.key }}"
+        matInput
+        [errorStateMatcher]="backendErrorsMatcher"
+      />
+      <mat-error
+        *ngIf="
+          additionalEmails.hasError('email', additionalEmailControl.key) ||
+          additionalEmails.hasError('pattern', additionalEmailControl.key)
+        "
+        i18n="@@register.invalidEmail"
+      >
+        Invalid email format
+      </mat-error>
+      <!--
+        <mat-error
+          *ngFor="
+            let error of getControlErrorAtFormLevel(
+              additionalEmailControl.value,
+              'allEmailsAreUnique'
+            )
+          "
+          i18n="@@register.additionalEmailsCanBeDuplicated"
+        >
+          Additional email cannot match primary email
+        </mat-error>
+      -->
+      <mat-error
+        *ngFor="
+          let error of getControlErrorAtFormLevel(
+            additionalEmailControl.value,
+            'backendErrors'
+          )
+        "
+      >
+        <app-backend-error [errorCode]="error"></app-backend-error>
+      </mat-error>
+    </mat-form-field>
+
+    <button
+      *ngIf="i == 0"
+      mat-icon-button
+      aria-label="info about names"
+      type="button"
+      [mdePopoverTriggerFor]="namesPopover"
+      mdePopoverTriggerOn="click"
+      #namesPopoverTrigger="mdePopoverTrigger"
+    >
+      <mat-icon>info</mat-icon>
+    </button>
+  </div>
+</ng-container>
+
+<a (click)="addAdditionalEmail()" class="mat-button-font">
+  <mat-icon>add</mat-icon>
+  <ng-container i18n="@@register.additionalEmailAdd"
+    >Add another email</ng-container
+  >
+</a>
+
+<mde-popover #namesPopover="mdePopover" [mdePopoverOverlapTrigger]="false">
+  <mat-card class="mat-elevation-z3 orcid-popover">
+    <ng-container i18n="@@register.additionalEmailInfo"
+      >Ensure that you never lose access to your ORCID record by registering all
+      of your email addresses on your account.</ng-container
+    >
+
+    <button
+      mat-button
+      color="primary"
+      aria-label="close"
+      (click)="namesPopoverTrigger.closePopover()"
```

Comment:
@leomendoza123 throws an error in the console when clicked it

Reply:
I have created this issue to solve this one https://github.com/ORCID/orcid-angular/issues/182

Reply:
This doesn't produce any usability limitation